Taskfile.yml

```
@@ -0,0 +1,191 @@
+      - doppler run -- python3 -m gunicorn --workers=2 --threads=2 core.wsgi:application --bind 0.0.0.0:{{or .CLI_ARGS 8080}}
+
+  schema-check:
+    dir: "{{.TASKFILE_DIR}}"
+
+    desc: Check Models and Serializers for schema compliance with OpenAPI .0 Standards and fail on warnings
+    cmds:
+      - doppler run --  python3 run/manage.py spectacular  --file schema/schema$(date +"%Y-%m-%d").yaml --validate
+    internal: true
+
+  schema:
+    deps: [ schema-check ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)